pkg/apis/deployment/v1/image_info.go

```diff
@@ -39,13 +40,17 @@ func (i *ImageInfo) String() string {
 		return "undefined"
 	}
 
-	e := "Community"
+	license := []rune(i.License())
+	license[0] = unicode.ToTitle(license[0])
 
+	return fmt.Sprintf("ArangoDB %s %s (%s)", string(license), string(i.ArangoDBVersion), i.Image)
+}
+
+func (i *ImageInfo) License() string {
```

Comment:
You are now changing public API interface which can break client backward-compatibility

Reply:
Done. Changes in v1 api reverted.